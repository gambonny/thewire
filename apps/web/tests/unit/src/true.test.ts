describe.concurrent('a test', () => {
  it('matches the snapshot', ({ expect }) => {
    expect.assertions(1)
    expect(1).toMatchSnapshot()
  })
})
