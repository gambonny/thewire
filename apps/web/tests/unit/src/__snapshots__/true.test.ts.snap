// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`pandaCSS Preset > matches the snapshot 1`] = `1`;
